docs/luigi-client-api.md

```diff
@@ -30,15 +30,148 @@ Returns **{idToken: [string](https://developer.mozilla.org/docs/Web/JavaScript/R
+```javascript
+LuigiClient.linkManager().fromContext('project').navigate('/settings')
+```
+
+Returns **[linkManager](#linkmanager)** link manager instance.
+
+### fromClosestContext
+
+Sets the current navigation context, which is then used by the navigate function. This has to be a parent navigation context, it is not possible to go to child navigation contexts.
+
+#### Examples
+
+```javascript
+LuigiClient.linkManager().fromClosestContext().navigate('/users/groups/stakeholders')
+```
+
+Returns **[linkManager](#linkmanager)** link manager instance.
+
+### withParams
+
+Sends node parameters to the route, which are then used by the navigate function. Use it optionally in combination with any of the navigation functions and receive it as part of the context object in Luigi Client.
+
+#### Parameters
+
+-   `nodeParams` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** 
+
+#### Examples
+
+```javascript
+LuigiClient.linkManager.withParams({foo: "bar"}).navigate("path")
+
+// Can be chained with context settings functions like this:
+LuigiClient.linkManager.fromContext("currentTeam").withParams({foo: "bar"}).navigate("path")
+```
+
+Returns **[linkManager](#linkmanager)** link manager instance.
+
+### hasBack
+
+Checks if there are one or more preserved views. Can be used to show a back button.
+
+Returns **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** a boolean with the information if there is a preserved view available to which a user can return.
+
+### goBack
+
+Discards the active view and navigates back to the last visited view (preserved view), if a preserved view was set before.
+
+#### Parameters
+
+-   `goBackValue` **any** data that is handed over in `goBackContext` field to the last visited view
+
+#### Examples
+
+```javascript
+LuigiClient.linkManager().goBack({ foo: 'bar' });
+LuigiClient.linkManager().goBack(true);
+```
+
+## uxManager
+
+Use the UX Manager to manage the appearance in Luigi.
+
+### showLoadingIndicator
+
+Adds a backdrop with a loading indicator for the micro front-end frame. This overrides the [loadingIndicator.enabled](navigation-configuration.md#nodes) setting.
+
+### hideLoadingIndicator
+
+Removes the loading indicator. Use it after calling [showLoadingIndicator()](#showLoadingIndicator) or to hide the indicator when you use the [loadingIndicator.hideAutomatically: false](navigation-configuration.md#nodes) node configuration.
+
+### addBackdrop
+
+Adds a backdrop to block the top and side navigation. It is based on Fundamental UI Modal, which you can use in your micro front-end to achieve the same behavior.
+
+### removeBackdrop
+
+Removes the backdrop.
+
+# Node parameters are defined like URL query parameters but with a specific prefix allowing them to be passed to the micro front-end view.  The default prefix is **~** and it is used in the following example: `https://my.luigi.app/home/products?~sort=asc~page=3`.
```

Comment:
Something went wrong here, I guess an unresolved merging conflict - below we can find the same document generated for the second time.

Reply:
Fixed.